src/pages/Swap/AdvancedSwapMode/Chart/ChartLoader.component.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react'
+
+import { FallbackLoader } from '../../../../components/Loader/FallbackLoader'
+import { AbsoluteWrapper } from './Chart.styles'
+
+const THREE_SECONDS = 3000
+
+export function ChartLoader({ symbol }: { symbol?: string }) {
+  const [isLoading, setIsLoading] = useState(true)
+
+  useEffect(() => {
+    setIsLoading(true)
+    const timeout = setTimeout(() => setIsLoading(false), THREE_SECONDS)
```

Comment:
does this charLoader always take 3 seconds?

Reply:
Yes, the chart needs to load at least 3 seconds